### ISSUE TITLE
Only send lines on screen

### ISSUE THREAD

**Issue by walseb** [emacs-languagetool/flycheck-languagetool#4]
Many thanks for this package! My only issue with it so far is that currently it sends the entire buffer to languagetool, which means my 6000 line org files will basically never be checked and I will always have 100% CPU while having them open. It would be great if you could make it only send the contents on screen, current line, etc. Flyspell already does this although it gets away with only sending the current word since it's only a spell-checker

**Comment by jcs090218** [comment]
Good idea! I may not have time to implement this feature recently but I would keep this in mind! 👍 Thanks! 😄

**Comment by mavit** [comment]
#19 enables you to narrow a large buffer to only check the narrowed section.  Not quite what was asked for, I know, but should help somewhat.
